src/changes-backwards-compatible.apib

```diff
@@ -42,3 +44,49 @@ We list all backwards-compatible additions here. These are currently available i
 
 - We now also return a `currency_exchange_rate` on all `.info` endpoints where we support multi-currency.  
   That is `quotations.info`, `invoices.info` and `creditNotes.info`.
+
+#### February 2019
+
+- We now also return the `invoicing_method` of a milestone. Both in `milestones.list` and `milestones.info`.
+
+- We now expose the invoiceability of time trackings and running timers. You can now use the `invoiceable` property on the following endpoints:
+  - `timeTracking.list`  
+  - `timeTracking.info`  
+  - `timeTracking.add`  
+  - `timeTracking.update`  
+  - `timers.current`  
+  - `timers.start`  
+  - `timers.update`  
+  We also return an `invoiceable` preference in the `users.me` endpoint that exposes whether a user prefers to track time by default as invoiceable or not.
```

Comment:
B: This adds the sentence to the bullet for `timers.update`, which shouldn't be the case.

Reply:
Fixed in #427.